templates/web/base/auth/generate_token.html

```diff
@@ -0,0 +1,50 @@
+[%
+SET bclass = 'authpage';
+SET bclass = 'fullwidthpage' IF password_changed;
+INCLUDE 'header.html', title = loc('Change password'), bodyclass = bclass
```

Comment:
This title and bits before it don't seem right :)
I'd go with fullwidthpage always I think.